machine/drivers/virtualbox.md

```diff
@@ -80,6 +81,7 @@ upper bound of `192.168.24.254`.
 | `--virtualbox-no-share`              | `VIRTUALBOX_NO_SHARE`              | `false`                  |
 | `--virtualbox-no-dns-proxy`          | `VIRTUALBOX_NO_DNS_PROXY`          | `false`                  |
 | `--virtualbox-no-vtx-check`          | `VIRTUALBOX_NO_VTX_CHECK`          | `false`                  |
+| `--virtualbox-share-folder`          | `VIRTUALBOX_SHARE_FOLDER`          | `/os/users:users`        |
```

Comment:
Is `/os/users:users` supposed to be literal? Couldn't find any hits in the source repo and I wasn't aware `/os/users` was a thing. Would `~:users` be okay?